compiler/main/checks.cpp

```diff
@@ -700,6 +703,86 @@ checkAutoCopyMap()
   }
 }
 
+static FnSymbol* findUserInitEq(AggregateType* at) {
+  for_alive_in_Vec(FnSymbol, fn, gFnSymbols) {
+    if (fn->name == astrInitEquals &&
+        !fn->hasFlag(FLAG_COMPILER_GENERATED) &&
+        fn->numFormals() >= 2) {
```

Comment:
Do we allow `init=` with more than two arguments, or do we catch them elsewhere?

Reply:
I meant to just check `numFormals >= 1` - I just didn't want to cause core dumps for erroneous `init=` here.

For `init=` with more arguments, I'd expect something like a default argument would be allowed. But that's not really the main issue in this PR. This routine in particular only exists for reporting error messages.